solidity/contracts/bridge/Bridge.sol

```diff
@@ -478,13 +495,43 @@ contract Bridge is Ownable, EcdsaWalletOwner {
         wallets.registerNewWallet(ecdsaWalletID, publicKeyX, publicKeyY);
     }
 
-    // TODO: Documentation.
+    /// @notice A callback function that is called by the ECDSA Wallet Registry
+    ///         once a wallet heartbeat failure is detected.
+    /// @param ecdsaWalletID Wallet's unique identifier.
+    /// @param publicKeyX Wallet's public key's X coordinate.
+    /// @param publicKeyY Wallet's public key's Y coordinate.
+    /// @dev Requirements:
+    ///      - The only caller authorized to call this function is `registry`
+    ///      - Wallet must be in Live state
     function __ecdsaWalletHeartbeatFailedCallback(
+        /* solhint-disable-next-line no-unused-vars */
```

Comment:
Instead of disabling solhint, I think we can do something like:
```
function __ecdsaWalletHeartbeatFailedCallback(
        bytes32,
        bytes32 publicKeyX,
        bytes32 publicKeyY
    ) external override {
```

and then, we should not even need to explain unused `ecdsaWalletID` in `@param`s.

Reply:
See c7e3a122